requirements/deploy.txt

```diff
@@ -344,15 +344,15 @@ requests-oauthlib==1.3.1
     # via
     #   -r requirements/pip.txt
     #   django-allauth
-requests-toolbelt==0.10.1
+requests-toolbelt==1.0.0
```

Comment:
Nothing relevant to us https://github.com/requests/toolbelt/blob/1.0.0/HISTORY.rst#100----2023-05-01